xtask/public-api/.gitattributes

```diff
@@ -0,0 +1 @@
+/*.txt binary
```

Comment:
I'm not a fan. This suppresses textual diffs when an API changes 😢

Reply:
It does, but you can still see it, I think. It's really upsetting when you're searching through code and the results are a lot of these .txt files.

Is there a way to get diffs while still ignoring these files in `git grep`?

Reply:
`git grep "NetlinkError" -- ':!*.txt'`

Reply:
Sorry, my question was too specific. I also want my editor to ignore these files.

Reply:
If it's vscode (or similar) we can add to the workspace settings:
```json
  "search.exclude": {
    "xtask/public-api/*.txt": true
  }
  ```

Reply:
I guess that's the best we can do. Done.